### ISSUE TITLE
Documentation on how to build adito ?

### ISSUE THREAD

**Issue by xbeaudouin** [toepi/adito#16]
Hi,

I am trying to make this adito working and maybe contribute in the future.

How to build (ok I know mvn package), and how to install/run it ? 

Since pointer to documentation on Sourceforge doesn't exist anymore :(

**Comment by toepi** [comment]
Hi,

nice to see that somebody is interessting in do some work for this project. I hope found more time to do it also ...

Sorry for the broken link.

At first build adito-distributions with `mvn -Pbase,extension` after that you have a zip wich contains every thing to run the server. 

Now the tricky part:
after unpack the zip go into the directory and execute `ant install` point your browser to the URL, after finished the setup you should be able to start the Server with `ant console` or `ant start-using-java`

The agent is currently broken it is not possible to use any extension which open a tunnel.

**Comment by kidhasmoxy** [comment]
Is the agent broken due to the lack of a object cert, or is it broken in your branch? The original r706 from SF is usable if you compile it with a JKS that has a valid (CA signed) object signing cert.

**Comment by toepi** [comment]
@kidhasmoxy: I don't understand your question, sry.

You can build the agent, I think you can sign the applet. But currently I have a bug the prevent you to use plugins which open a dynamic tunnel. Have found it but not time do do further investigations on it.

I think current trunk at SFO is also broken? But I have try it a long time ago, so not really sure. But I know that I started from a tagged revision to have a working starting point.

**Comment by kidhasmoxy** [comment]
There's a working version of the agent in the r760 release from sourceforge, which works as long as you build it with a proper object cert. That's all temporary anyway, with the war on NPAPI, and the fact that this applet probably won't see the updates it needs to work in the new world.

**Comment by toepi** [comment]
Yes applets (Thanks Google, for this stupid step) are deprected but currently no one as time to replace all the code to use HTML5 (websockets)? If you are interesst in use rdp/vnc only - change to Guacamole (http://guac-dev.org/).
